contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
++
+* Implementations of various _probability distributions_ are used in generative modeling, such as Gaussian distributions, categorical distributions, and multinomial distributions. Probability distributions are fundamental for modeling uncertainty and generating diverse outputs.
++
+* Offer _evaluation metrics_ for assessing the performance of generative AI models, such as inception score, Frechet Inception Distance (FID), perplexity, and BLEU score. These metrics provide quantitative measures of the quality, diversity, and realism of generated outputs.
++
+* Support _parallelization techniques_ and _GPU acceleration_ to speed up the training and inference process, especially for large-scale generative models that require significant computational resources.
++
+* Provide tools for deploying trained generative _AI models_ into production environments, including integration with deployment platforms, model serialization formats, and optimization techniques for inference speed and resource efficiency.
+
+== Development Environments
+
+. *Many developers opt for lightweight integrated developer environments (IDEs), rather than the full-fledged IDE. What lightweight IDEs are popular for pass:[C++] development?*
++
+There are several popular options for both Windows and Linux. https://www.jetbrains.com/clion/[CLion], developed by JetBrains, is a cross-platform IDE that offers advanced code analysis, refactoring tools, and integration with the CMake build system, which is commonly used in pass:[C++] projects.
++
+https://www.gnu.org/software/emacs/[GNU Emacs] and https://www.vim.org/[Vim] are highly configurable and popular among developers who prefer a more minimalistic environment. They offer powerful features for editing code, and many plugins are available to enhance development workflows.
++
+https://www.qt.io/product/development-tools[Qt Creator] provides features like code completion, syntax highlighting, and debugging support for pass:[C++] and Qt (projects that use the Qt framework).
++
+https://www.sublimetext.com/index2[Sublime Text] is a lightweight yet powerful text editor known for its speed and simplicity. It offers features like syntax highlighting, multiple selections, and a wide range of plugins for enhancing functionality, including support for pass:[C++] development.
++
+https://atom-editor.cc/[Atom] is an open-source text editor developed by GitHub. It's highly customizable and extensible through packages, and provides features like syntax highlighting, auto-completion, and project navigation.
++
+There are many other tools, https://code.visualstudio.com/Download[Visual Studio Code] being a popular versatile code editor that can be extended and customized with various extensions.
+
+== Existing Boost Libraries
```

Comment:
Are these questions contributors have? None of that ever crossed my mind.

Reply:
They are based on real questions/comments, particularly about the steep learning curve. The idea is to provide a new library developer with upper and lower boundaries of complexity that they might use to guide the complexity of their submission. However, i can remove or reword if this isn't adding value.